### DISCUSSION TITLE
Update error at Esphome

### DISCUSSION THREAD

**Discussion by Toolmaster30** [tteck/Proxmox#3327] [General] [answered]
Hello, since the last Debian 12 update I can no longer update Esphome. The following error occurs:

`✓ Stopped ESPHome
 | Updating ESPHome   
[ERROR] in line 66: exit code 0: while executing command pip3 install -U esphome &> /dev/null`

Georg

**Comment by divemasterjm** [comment, marked as answer]
as ttech said
If you have manually updated to Debian 12.6, execute `rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED` in the LXC console.
The Proxmox VE LXC Updater and Proxmox VE Cron LXC Updater has modified for Dedian 12.6

this worked for me

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/3300

**Reply by divemasterjm** [reply]
trying to update to esphome 2024.8 results in after succesfull update
 esphomeDashboard.service - ESPHome Dashboard
     Loaded: loaded (/etc/systemd/system/esphomeDashboard.service; enabled; preset: enabled)
     Active: failed (Result: exit-code) since Wed 2024-08-21 09:56:37 CEST; 58s ago
   Duration: 67ms
    Process: 219 ExecStart=/usr/local/bin/esphome /root/config/ dashboard (code=exited, status=2)
   Main PID: 219 (code=exited, status=2)
        CPU: 66ms

Aug 21 09:56:37 esphome systemd[1]: esphomeDashboard.service: Scheduled restart job, restart counter is at 5.
Aug 21 09:56:37 esphome systemd[1]: Stopped esphomeDashboard.service - ESPHome Dashboard.
Aug 21 09:56:37 esphome systemd[1]: esphomeDashboard.service: Start request repeated too quickly.
Aug 21 09:56:37 esphome systemd[1]: esphomeDashboard.service: Failed with result 'exit-code'.
Aug 21 09:56:37 esphome systemd[1]: Failed to start esphomeDashboard.service - ESPHome Dashboard.

**Reply by tteck** [reply]
No issue here

```
    ___________ ____  __  __                   
   / ____/ ___// __ \/ / / /___  ____ ___  ___ 
  / __/  \__ \/ /_/ / /_/ / __ \/ __ `__ \/ _ \
 / /___ ___/ / ____/ __  / /_/ / / / / / /  __/
/_____//____/_/   /_/ /_/\____/_/ /_/ /_/\___/ 
                                               
 ✓ Stopped ESPHome
 ✓ Updated ESPHome
 ✓ Started ESPHome
 ✓ Updated Successfully
root@esphome:~# 
```

**Reply by divemasterjm** [reply]
> No issue here
> 
> ```
>     ___________ ____  __  __                   
>    / ____/ ___// __ \/ / / /___  ____ ___  ___ 
>   / __/  \__ \/ /_/ / /_/ / __ \/ __ `__ \/ _ \
>  / /___ ___/ / ____/ __  / /_/ / / / / / /  __/
> /_____//____/_/   /_/ /_/\____/_/ /_/ /_/\___/ 
>                                                
>  ✓ Stopped ESPHome
>  ✓ Updated ESPHome
>  ✓ Started ESPHome
>  ✓ Updated Successfully
> root@esphome:~# 
> ```

I have same, but service stoped after update with the error I've posted

**Comment by Toolmaster30** [comment]
Thank you very much. Works perfectly.
